### ISSUE TITLE
Holograms 230 - Spatial Mapping - surfaceObserver not observing in Chapters 1 & 2

### ISSUE THREAD

**Issue by roelkok** [Microsoft/HolographicAcademy#35]
If I follow _Chapter 1: Scanning_ and _Chapter 2: Visualization_ the project doesn't show anything in my HoloLens. The problem seems that `StartObserver` doesn't get called in the `SpatialMappingManager`. So no data is acquired and nothing will be displayed.

In _Chapter 3: Processing_ there will be visual output once the `PlaySpaceManager` has been updated, because it will enter scanning mode if not enough surfaces have been found.

**Comment by chriskurzeja** [comment]
Agreed. I managed to make the example work again by porting [code from the Toolkit version of SpatialMappingManager](https://github.com/Microsoft/HoloToolkit-Unity/blob/master/Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs). This fixes the issue with no mesh being shown in _Chapter 1_ and _Chapter 2_.

Specifically I added the autoStartObserver property and Start method back into the SpatialMappingManager address this issue according to the implementation in the Toolkit.

```csharp
       [Tooltip("Determines if the surface observer should be automatically started.")]
        public bool autoStartObserver = true;
```

```csharp
        // Use for initialization.
        private void Start()
        {
            if (autoStartObserver)
            {
                StartObserver();
            }
        }
```

Hopefully this is an easy fix to actually apply.
